src/fetcher/get-initial-props.ts

```diff
@@ -60,15 +60,23 @@ export async function getInitialProps(
       })
       props.res?.end()
       // We redirect here so the component won't be actually rendered
-      return { origin: '', pageData: { kind: 'error' } }
+      return {
+        origin: '',
+        pageData: { kind: 'error', errorData: { code: 200 } },
+      }
     }
 
     if (fetchedData.error) {
-      props.res!.statusCode = 404
+      const code = fetchedData.error.includes(
+        "Cannot read property 'path' of null"
```

Comment:
Not happy about that. Can we cover that in the fetcher itself (Namely checking if uuid / whatever is null)? The actual error message might change depending on which node version Vercel is running on.

Reply:
The request throws an exception with this message: 
![grafik](https://user-images.githubusercontent.com/13507950/88680778-28e4cb00-d0f1-11ea-9415-513f7d7f0269.png)

Can the API avoid throwing this error and just pass uuid as null instead? Otherwise the frontend is not much able to tell what happened.

Reply:
Ah, didn't know that this happens at API level. My bad, gonna look into this.

Reply:
No problem, thanks for investigating this.